gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenViewModel.java

```diff
@@ -87,7 +86,14 @@
                         .doOnError(this::onAddFeatureError)
                         .onErrorResumeNext(Single.never())) // Prevent from breaking upstream.
             .observeOn(schedulers.ui())
-            .subscribe(this::showBottomSheet));
+            .subscribe(this::addNewObservation));
```

Comment:
Cool, thanks! We really needed this! One caveat, we should only direct to add observation if the layer in which the feature lives has a form defined. Can we check that first?

Reply:
Done